Tests/SourceKitLSPTests/ExecuteCommandTests.swift

```diff
@@ -46,19 +46,28 @@ final class ExecuteCommandTests: XCTestCase {
 
     let request = ExecuteCommandRequest(command: command.command, arguments: command.arguments)
 
+    let expectation = self.expectation(description: "Handle ApplyEditRequest")
+    let applyEditTitle = ThreadSafeBox<String?>(initialValue: nil)
+    let applyEditWorkspaceEdit = ThreadSafeBox<WorkspaceEdit?>(initialValue: nil)
+
     testClient.handleSingleRequest { (req: ApplyEditRequest) -> ApplyEditResponse in
+      applyEditTitle.value = req.label
+      applyEditWorkspaceEdit.value = req.edit
+      expectation.fulfill()
+
       return ApplyEditResponse(applied: true, failureReason: nil)
     }
 
-    let result = try await testClient.send(request)
+    _ = try await testClient.send(request)
 
-    guard case .dictionary(let resultDict) = result else {
-      XCTFail("Result is not a dictionary.")
-      return
-    }
+    try await fulfillmentOfOrThrow([expectation])
 
+    let label = try XCTUnwrap(applyEditTitle.value)
+    let edit = try XCTUnwrap(applyEditWorkspaceEdit.value)
+
+    XCTAssertEqual(label, "Localize String")
     XCTAssertEqual(
-      WorkspaceEdit(fromLSPDictionary: resultDict),
```

Comment:
Can we remove the `WorkspaceEdit(fromLSPDictionary:)` initializer and `WorkspaceEdit.encodeToLSPAny` now?

Reply:
I guess we can.